### ISSUE TITLE
Add Ignore URL Path Regex?

### ISSUE THREAD

**Issue by mcouillard** [newrelic/newrelic-php-agent#435]
As supported by the Ruby agent, can this agent be enhanced to skip transaction or even metric collection by URL path regex?
https://docs.newrelic.com/docs/apm/agents/ruby-agent/api-guides/ignoring-specific-transactions/

We're using Laminas (Zend) for MVC.

ElasticAPM offers this via `elastic_apm.transaction_ignore_urls`
https://www.elastic.co/guide/en/apm/agent/php/master/configuration-reference.html#config-transaction-ignore-urls

Having this allows us to skip certain ping / health checks that happen frequently for internal purposes, like load balancing.

**Comment by mcouillard** [comment]
Looks like this in-code solution does offer a solution:
https://docs.newrelic.com/docs/apm/agents/php-agent/php-agent-api/newrelic_ignore_transaction

But I was really seeking a config option outside of code in order to keep new relic specific logic outside of our codebases.

**Comment by mmenozzi** [comment]
Yes, moreover when PHP is executed in a PHP-FPM container with an health check configured to hit the PHP-FPM [ping.path](https://www.php.net/manual/en/install.fpm.configuration.php) is not possible to add a `newrelic_ignore_transaction()` call somewhere because it's PHP-FPM itself who generate the response to the ping request.

Usually the health check is configured to run frequently (for example every 10 seconds) and all those transactions are tracked by NewRelic so, on a low traffic application, is probably the "Most time consuming transaction" (it appears as a transaction named `/200`).

It would be very useful if we can exclude it with a NewRelic PHP Agent configuration.

**Comment by cpearcey** [comment]
Thank you for your feedback and for taking the time to share this suggestion. After reviewing this with the team, we've determined that this is not something we will be working on at this time as it doesn't align with our current product focus. We'll be closing this issue to keep our backlog manageable, but we appreciate your contribution to the community.
